src/platforms/javascript/guides/capacitor/troubleshooting.mdx

```diff
@@ -5,6 +5,38 @@ keywords: ["minimum version", "deployment target"]
 sidebar_order: 1000
 ---
 
+### Different versions for Sibling SDKs 
+
+Sentry Capacitor requires your app to use the same versions of @Sentry/Angular (or another Sibling SDK) that is refered by the peer dependency of Sentry Capacitor, to avoid any issues when installing packages or using the code on another machine, it's recommended to use the exact version of both sentry/capacitor and the sibling SDK. 
```

Comment:
This seems repetitive. Suggest:

```suggestion
To avoid any issues when installing packages or using the code on another machine, you must use the same version of Sentry Capacitor and @Sentry/Angular (or any other sibling SDK) that is referred by the peer dependency of Sentry Capacitor in your app.  
```